### PR TITLE
feature/localmaven

### PR DESCRIPTION
**Pull request by SalmanAshraf70**
- Setup local maven library
- added localmaven() in settings.gradle
- Update the code in publish_local.gradle

### CODE REVIEW

File: app/src/main/java/com/moneytransfer/demo/MainActivity.kt [hunk 1]
```diff
@@ -7,6 +7,7 @@ import androidx.navigation.findNavController
 import androidx.navigation.ui.AppBarConfiguration
 import androidx.navigation.ui.setupActionBarWithNavController
 import androidx.navigation.ui.setupWithNavController
+import com.moneytransfer.core.SampleCore
```
Comment (Srikanth-Aleph) [inline app/src/main/java/com/moneytransfer/demo/MainActivity.kt]: Please remove this unused Dependency

File: build.gradle [hunk 2]
```diff
@@ -3,6 +3,7 @@ buildscript {
     repositories {
         mavenLocal()
         google()
+        mavenLocal()
```
Comment (Srikanth-Aleph) [inline build.gradle:5]: Remove redundant mavenLocal()

File: custom-http/build.gradle [hunk 3]
```diff
@@ -1,8 +1,8 @@
 plugins {
     id 'com.android.library'
     id 'kotlin-android'
+    id 'maven-publish'
```
Comment (Srikanth-Aleph) [inline custom-http/build.gradle]: I believe it should be in publish_local.gradle file. 
is there any specific reason for adding it here? :)

**Review by Srikanth-Aleph — CHANGES_REQUESTED** [review]

File: custom-http/build.gradle (hunk 3 above)
Comment (SalmanAshraf70) [reply custom-http/build.gradle]: We are publishing the whole module so all plugins should be in one place

Comment (Srikanth-Aleph) [reply custom-http/build.gradle]: I suggest to keep it in the publish_local.gradle file and use it while publishing any other modules also.

File: publish_local.gradle [hunk 4]
```diff
@@ -1,3 +1,5 @@
+apply plugin: 'maven-publish'
```
Comment (Srikanth-Aleph) [inline publish_local.gradle]: Great!

**Review by Srikanth-Aleph — APPROVED** [review]

**Pull request merged**